lib/python/qmk/cli/cformat.py

```diff
@@ -26,7 +26,8 @@ def cformat(cli):
     else:
         for dir in ['drivers', 'quantum', 'tests', 'tmk_core']:
             for dirpath, dirnames, filenames in os.walk(dir):
-                if 'tmk_core/protocol/usb_hid' in dirpath:
+                ignores = ['tmk_core/protocol/usb_hid', 'quantum/template']
+                if any(i in dirpath for i in ignores):
                     continue
```

Comment:
Instead of skipping every dir/file individually, this will remove the ignorable(?) directory from the list and os.walk will not ascend into it.
```
   for dn in dirnames: 
       if Path(dirpath) / dn in ignores: 
           dirnames.remove(dn) 
```